catalog/task/update-ocp-tag/0.1/update-ocp-tag.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: update-ocp-tag
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to update pull-spec tag with
+    valid OCP version from get-ocp-version task.
+  params:
+    - name: fromIndex
+      description: The source Index image (catalog of catalogs) FBC fragment
+      type: string
+    - name: targetIndex
+      description: Index image (catalog of catalogs) the FBC fragment will be added to
+      type: string
+    - name: binaryImage
+      description: OCP binary image to be baked into the index image
+      default: ""
+      type: string
+    - name: ocpVersion
+      description: Stored OCP version from get-ocp-version task result
+      type: string
+  results:
+    - name: updated-fromIndex
+      description: Store OCP version number from given Image
+    - name: updated-targetIndex
+      description: Store OCP version number from given Image
+    - name: updated-binaryImage
+      description: Store OCP version number from given Image
+  steps:
+    - name: update-ocp-tag
+      image: quay.io/hacbs-release/release-utils:4d8649dbb2b626f5fe9f4ff83c1bc3be268fad31
+      script: |
+       #!/usr/bin/env sh
+       set -eux
+
+        fromIndex="$(params.fromIndex)"
```

Comment:
I am confused why we do this instead of just use the param values directly on lines 59-61